src/ledger/AccountFrame.cpp

```diff
@@ -353,6 +356,47 @@ AccountFrame::countObjects(soci::session& sess)
     return count;
 }
 
+uint64_t
+AccountFrame::countObjects(soci::session& sess,
```

Comment:
I think that you could use LedgerRange class here (from catchup-at PR, now in master).
countObjects(sess, int, int) looks cryptic from API point of view
countObjectsInRange(sess, int, int) is better
but
countObjects(sess, LedgerRange) is best ;)